### PR TITLE
Document test runner

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to openfisca/openfisca-core#429

[Python test runner](http://openfisca.readthedocs.io/en/latest/test_runner.html)
[Command line tool]( http://openfisca.readthedocs.io/en/latest/openfisca-run-test.html)

### CODE REVIEW

**Comment by fpagnoux** [comment]
I was just deploying it ;)

File: source/openfisca-run-test.rst [hunk 1]
```diff
@@ -0,0 +1,229 @@
+==================
+openfisca-run-test
+==================
+
+.. argparse::
+   :module: openfisca_core.scripts.run_test
+   :func: build_parser
+   :prog: openfisca-run-test
+
+Exemples
```
Comment (MattiSG) [inline source/openfisca-run-test.rst]: Examples

File: source/openfisca-run-test.rst [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+**Command line:**
+
+.. code-block:: shell
+
+  openfisca-run-test -c openfisca_france test.yaml
+  # Success
+
+  openfisca-run-test test.yaml
+  # Success : the country package is automatically detected.
+  # May fail if several country packages are installed on your environment.
```
Comment (MattiSG) [inline source/openfisca-run-test.rst]: on → in

File: source/openfisca-run-test.rst [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+**Command line:**
+
+.. code-block:: shell
+
+  openfisca-run-test -c openfisca_france test.yaml
+  # Success
+
+  openfisca-run-test test.yaml
+  # Success : the country package is automatically detected.
```
Comment (MattiSG) [inline source/openfisca-run-test.rst]: No space before colons in English.

File: source/openfisca-run-test.rst (hunk 2 above)
Comment (MattiSG) [inline source/openfisca-run-test.rst]: You need to tell me how to solve the issue: `In that case, specify which package to use with the --country_package option`.

File: source/openfisca-run-test.rst [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+**Command line:**
+
+.. code-block:: shell
+
+  openfisca-run-test test_2.yaml
+  # Success : the second test is ignored
+
+  openfisca-run-test -f test_2.yaml
+  # Failure : the second test is executed and does not pass
```
Comment (MattiSG) [inline source/openfisca-run-test.rst]: No space before colon.

File: source/openfisca-run-test.rst [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+
+.. code-block:: shell
+
+  openfisca-run-test test_4.yaml
+  # Failure : the third test does not pass
+
+  openfisca-run-test -n openfisca test_4.yaml
+  # Success : the third test is not executed, as it doesn't contain the word 'openfisca'
+
+Note that if a test file name contains the name filter, all the inside tests will be executed.
```
Comment (MattiSG) [inline source/openfisca-run-test.rst]: “All the tests inside” or “all the inner tests”.

**Comment by fpagnoux** [comment]
Feedback taken into account in https://github.com/openfisca/openfisca-core/pull/429/files

**Comment by MattiSG** [comment]
@fpagnoux  Should we close this if everything is moved to openfisca/openfisca-core#429?

**Comment by fpagnoux** [comment]
>Couldn't we use Markdown?

My understanding is the following : 

- Sphinx is the reference to write documentation in python.
- Sphinx and rST together offer very useful features, such as auto generation of doc, or cross-referencing.
- There are [some ways](http://blog.readthedocs.com/adding-markdown-support/) to write doc in markdown with Sphinx, but this seems more recent, not well-spread ,and for the features mentioned above, you still have to use rST (and I don't know exactly how well you can mix both in the same file, for instance).

For this reason, I gave up markdown at least for now. If we want to write more text, and that rST become a pain, we can consider using the extension, but for now most of what we use is the autodoc.

@cbenz probably knows more about all this.

**Comment by fpagnoux** [comment]
>@fpagnoux  Should we close this if everything is moved to openfisca/openfisca-core#429?

Yes. Was just answering one last question ;)L

**Pull request closed without merging**